pkgs/development/libraries/pipewire/default.nix

```
@@ -62,14 +62,13 @@
 , openssl
 , rocSupport ? true
 , roc-toolkit
-, x11Support ? true
+, x11Support ? true, ffado
 , libcanberra
 , xorg
 , mysofaSupport ? true
 , libmysofa
 , tinycompress
-, ffadoSupport ? stdenv.buildPlatform.canExecute stdenv.hostPlatform
-, ffado
+, ffadoSupport ? x11Support && stdenv.buildPlatform.canExecute stdenv.hostPlatform
```

```suggestion
, ffadoSupport ? x11Support && stdenv.buildPlatform.canExecute stdenv.hostPlatform
, ffado
```
lets keep this like this, the package should be near the option